### DISCUSSION TITLE
Fallback to standard language if desired language doesn't exist in tiles?

### DISCUSSION THREAD

**Discussion by christian2701** [maplibre/maplibre-gl-js#2957] [Q&A] [answered]
Hello dear Maplibre community,

I am currently working on a new map style with self-created tiles that will be usable in different languages. 

Is it necessary to export each of the desired languages or is there a possibility to a fallback language in the style.json?

Example:
Street in France "Quai Jacques Chirac" would currently only exist as "name" and as "name:fr". As soon as I want to display the street in Spanish, it would disappear because "name:es" is not available in the data. I would like to use "name" as a fallback for this use case. Is this possible?

I fear that generating 25 languages for all road labels could be a performance problem (generating and rendering).

Thank you!

**Comment by HarelM** [comment, marked as answer]
I think you can use `coalesce` expression I believe.

**Reply by christian2701** [reply]
Thank you very much for the quick feedback. Coalesce and also case work as expected. I will now test which expression makes more sense for our use case.

**Comment by wipfli** [comment]
Or something with `case` maybe:
```json
[
"case",
["has", "name:fr"],
["get", "name:fr"],
["get", "name"]
]
```

**Reply by christian2701** [reply]
Thank you very much for the quick feedback. Coalesce and also case work as expected. I will now test which expression makes more sense for our use case.
